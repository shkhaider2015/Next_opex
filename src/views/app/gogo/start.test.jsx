import React from 'react';
import renderer from 'react-test-renderer';
import Start from './start';
import { IntlProvider } from 'react-intl';
import AppLocale from '../../../lang';

const locale = 'en';
const currentAppLocale = AppLocale[locale];

describe("Start", () => {
    it("Should render chart", () => {
        const component = renderer.create(
        <IntlProvider locale={currentAppLocale.locale}
            messages={currentAppLocale.messages}>
            <Start/>
        </IntlProvider>).toJSON();
        expect(component).toMatchSnapshot();
    })
});
