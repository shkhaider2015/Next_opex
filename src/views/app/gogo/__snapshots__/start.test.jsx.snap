// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Start Should render chart 1`] = `
Array [
  <div
    className="row"
  >
    <div
      className="col-12"
    >
      <div
        className="separator mb-5"
      />
    </div>
  </div>,
  <div
    className="row"
  >
    <div
      className="mb-4 col-12"
    >
      <p>
        <span>
          Start
        </span>
      </p>
      <button
        aria-label={null}
        className="btn btn-secondary btn-lg"
        onClick={[Function]}
        type="button"
        variant="secondary"
      >
        5min
      </button>
      <button
        aria-label={null}
        className="btn btn-secondary btn-lg"
        onClick={[Function]}
        type="button"
        variant="secondary"
      >
        10min
      </button>
      <button
        aria-label={null}
        className="btn btn-secondary btn-lg"
        onClick={[Function]}
        type="button"
        variant="secondary"
      >
        Interactive
      </button>
    </div>
  </div>,
  <div
    className="row"
  >
    <div
      className="mb-34 col-12"
    />
  </div>,
  <div
    className="row"
  >
    <button
      aria-label={null}
      className="btn btn-secondary btn-lg"
      onClick={[Function]}
      type="button"
      variant="secondary"
    >
      Купить
    </button>
    <button
      aria-label={null}
      className="btn btn-secondary btn-lg"
      onClick={[Function]}
      type="button"
      variant="secondary"
    >
      Продать
    </button>
  </div>,
]
`;
